notebooks/407-person-tracking-webcam/407-person-tracking.ipynb

```diff
@@ -0,0 +1,616 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Person Tracking\n",
+    "\n",
+    "This notebook shows a person tracking scenario: it reads frames from an input video sequence, detects people in the frames, uniquely identify each one of them and track all of them until they leave the frame. We use the [Deep SORT](https://arxiv.org/abs/1703.07402) algorithm to perform object tracking, an extension to SORT (Simple Real time Tracker).\n",
+    "\n",
+    "## Detection vs Tracking\n",
+    "- In object detection, we detect an object in a frame, put a bounding box or a mask around it and classify the object. Note that, the job of the detector ends here. It processes each frame independently and identifies numerous objects in that particular frame. \n",
```

Comment:
what does it mean classify object here?
detection model which you use in notebook makes object prediction and classification as one step (single shot detector), but looks like it is description of 2 step detection like faster rcnn

Reply:
Hi @eaidova This part is used to educate our users what's the different between a detection task and a tracking task. ```put a bounding box or a mask around it and classify the object``` means we have to find the both the category and position of object in a detection task.